openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -878,6 +878,13 @@ void __kmp_create_worker(int gtid, kmp_info_t *th, size_t stack_size) {
     KMP_SYSFAIL("pthread_create", status);
   }
 
+#if defined(KMP_OS_LINUX) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 12)))
+  // Rename worker threads for improved debuggability
+  if (!KMP_UBER_GTID(gtid)) {
+    pthread_setname_np(handle, "openmp_worker");
```

Comment:
so other glibc version doesn't have this function or it doesn't work?

Reply:
`pthread_setname_np` is introduced in glibc 2.12, so yes, older version of glibc doesn't have this API.

Reply:
If this is the case, I'd suggest to use the CMake feature [`CheckSymbolExists`](https://cmake.org/cmake/help/latest/module/CheckSymbolExists.html#module:CheckSymbolExists).

Reply:
updated! :)